scripts/fig1-sample_disease_barchart.R

```diff
@@ -0,0 +1,121 @@
+# Generates a bar plot summarizing the diagnoses type in the ScPCA Portal 
+# The resulting plot is faceted by the diagnoses group 
+# Individual bars have different patterns based on the disease timing 
+
+# load any libaries 
+library(ggplot2)
+
+# Set up -----------------------------------------------------------------------
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# path to sample metadata stored on S3
+sample_metadata_s3 <- 's3://ccdl-scpca-data/sample_info/scpca-sample-metadata.tsv'
+
+# create a directory to store S3 files 
+local_s3_dir <- file.path(root_dir, "s3_files")
+fs::dir_create(local_s3_dir)
+
+# all metadata files 
+sample_info_dir <- file.path(root_dir, "sample-info")
+project_whitelist_file <- file.path(sample_info_dir, "project-whitelist.txt")
+diagnosis_groupings_file <- file.path(sample_info_dir, "diagnosis-groupings.tsv")
+disease_timing_file <- file.path(sample_info_dir, "disease-timing.tsv")
+
+# color palette
+diagnosis_group_palette <- file.path(root_dir, "palettes", "diagnosis-group-palette.tsv")
+
+# output files 
+plots_dir <- file.path(root_dir, "figures", "pngs")
+output_plot_file <- file.path(plots_dir, "Fig1-sample-summary.png")
+
+# Prep sample metadata ------------------------------------------------------
+
+# read in project whitelist and grouping metadata 
+project_whitelist <- readLines(project_whitelist_file)
+diagnosis_groupings_df <- readr::read_tsv(diagnosis_groupings_file) |>
+  dplyr::select(submitted_diagnosis, diagnosis_group)
+disease_timing_df <- readr::read_tsv(disease_timing_file)
+
+# Copy sample metadata 
+local_sample_metadata <- file.path(local_s3_dir, "scpca-sample-metadata.tsv")
+sync_call <- paste('op run -- aws s3 cp', sample_metadata_s3, local_sample_metadata, '--recursive', sep = " ")
```

Comment:
Something is funky here - it copies the file _as a directory_. This works for me instead - 

```suggestion
sync_call <- paste('op run -- aws s3 cp', sample_metadata_s3, local_s3_dir, sep = " ")
```

Reply:
Since I just saw this: `op run` should not be internal to a script: The user should invoke that on the wrapper script as needed.

Reply:
```suggestion
sync_call <- paste('aws s3 cp', sample_metadata_s3, local_s3_dir, sep = " ")
```